src/Nancy/Bootstrapper/NancyBootstrapperBase.cs

```diff
@@ -245,6 +245,8 @@ public void Initialise()
 
             this.ConfigureApplicationContainer(this.ApplicationContainer);
 
+            AppDomainAssemblyTypeScanner.UpdateTypes();

```

Comment:
Not sure about this one so bat signal for @NancyFx/most-valued-minions  @NancyFx/owners